configuration/configuration_blackbox_test.go

```diff
@@ -89,7 +89,7 @@ func TestGetKeycloakEndpointSetByUrlEnvVaribaleOK(t *testing.T) {
 func TestGetKeycloakEndpointAdminDevModeOK(t *testing.T) {
 	resource.Require(t, resource.UnitTest)
 	t.Parallel()
-	checkGetKeycloakEndpointOK(t, "http://sso.demo.almighty.io/auth/admin/realms/fabric8", config.GetKeycloakEndpointAdmin)
+	checkGetKeycloakEndpointOK(t, "http://sso.prod-preview.openshift.io/auth/admin/realms/fabric8", config.GetKeycloakEndpointAdmin)
```

Comment:
We don't need to hardcode the usage of dev mode sso as soon as we merge https://github.com/almighty/almighty-core/pull/1015